torch/_export/serde/schema.py

```diff
@@ -105,6 +110,16 @@ class SymIntArgument(_Union):
     as_name: str
     as_int: int
 
+# In most cases we will use the "as_name" field to store arguments which are
+# SymInts.
```

Comment:
Modify the comment to refer to symFloats?

Reply:
Yes it was one of the bullet point in the todo